atlite/cutout.py

```diff
@@ -618,6 +642,27 @@ def layout_from_capacity_list(self, data, col="Capacity"):
         )
         return data.to_xarray().reindex_like(self.data).fillna(0)
 
+    def layout_from_area_density(self, capacity_density, crs=3035):
+        """
+        Get a capacity layout from a uniform capacity density.
```

Comment:
I find the formulation "from density" a bit confusing. What do you think about `uniform_density_layout(...)` instead, to be consistent with the `uniform_layout(...)`?